dev-start.sh

```diff
@@ -104,12 +104,30 @@ changeNodeVersion() {
   fi
 }
 
+setupLocalKinesis() {
+  sleep 3s
+  # java sdk use CBOR protocol
+  # which does not work with localstack kinesis which use kinesislite
+  export AWS_CBOR_DISABLE=true
+  # dummy env variables for local kinesis
+  export AWS_ACCESS_KEY_ID=foobar
```

Comment:
could we use the `--profile` cli flag rather than env vars here?

Reply:
it does not matter :) those variables need to exists with whatever
This is more for AWS cli rather then for localstack on localhost so actual kinesis

Reply:
done